pkg/lib/form-layout.less

```diff
@@ -110,30 +111,38 @@
   }
 
   // Special considerations for widgets (and widget-like elements)
-  > input,
-  > textarea,
-  > select,
-  > .bootstrap-select,
-  > .dropdown,
-  > .combobox-container,
-  > fieldset,
-  > [role=group],
-  > .form-group,
-  > .btn-group,
-  > .checkbox-inline,
-  > .radio-inline {
-    // Vertically align elements with text
-    margin-top: -(@padding-small-vertical - @padding-base-vertical);
-    // Ensure widgets are a at least a minimum widget size
-    min-height: @widget-height;
-
-    // Some elements need special width considerations
-    // as PatternFly normally fixes the width
-    &:not(.ct-form-layout-relax) {
-      width: auto !important;
+  // This is a LESS mixin that will not be in the compiled CSS.
+  .widget-rules() {
+    > input,
+    > textarea,
+    > select,
+    > .bootstrap-select,
+    > .dropdown,
+    > .combobox-container,
+    > fieldset,
+    > [role=group],
+    > [data-field],
+    > .form-group,
+    > .btn-group,
+    > .checkbox-inline,
+    > .radio-inline {
+      // Vertically align elements with text
+      margin-top: -(@padding-small-vertical - @padding-base-vertical);
+      // Ensure widgets are a at least a minimum widget size
+      min-height: @widget-height;
     }
   }
 
+  &, > .ct-validation-wrapper {
+    .widget-rules();
+  }
+
+  // Some elements need special width considerations
+  // as PatternFly normally fixes the width
+  > :not(.ct-form-layout-relax) {
```

Comment:
I moved this to a top-level, but I'm not so happy with this, as there's an `!important`. I'd like to change this somehow, but don't want to rewrite all the selectors above.

I haven't found how to make it apply to the selectors above, but not the ones with the validation workaround sandwich.